### ISSUE TITLE
Enable pagination in blog view

### ISSUE THREAD

**Issue by aldrineeinsteen** [jakewies/hugo-theme-codex#78]
Add configuration support in the blog section.

Am thinking about

[params]
  [params.blog]
     ...
     pagination = true
     paginate = 5

**Comment by jakewies** [comment]
@kentnek this is a feature I'd like to have a bigger discussion about. After some thought, I think it would be best to a have a default pagination setup for the blog list, and the default can be overridden using configuration such as @aldrineeinsteen has described.

What are your thoughts?

**Comment by kentnek** [comment]
I don't mind having a pagination, as long as the styling fits our overall look of the theme. I don't really dig the bright red page button in @aldrineeinsteen's PR, which instead can be a semi-transparent grey box like this existing Prev/Next button:

![test](https://user-images.githubusercontent.com/7024160/87436160-3e50f400-c61f-11ea-8a3f-f24f2d137cd3.gif)


Also, the config should also be site-wide, not page-wide, as [discussed earlier](https://github.com/jakewies/hugo-theme-codex/issues/73#issuecomment-657630761).

**Comment by jakewies** [comment]
Agreed on all accounts. The styles must fit the theme. And the config should exist in config.toml site wide. @aldrineeinsteen if you need help with styling let us know.

**Comment by miguelbemartin** [comment]
Is it possible to paginate the blog? I don't find the way...

**Comment by jakewies** [comment]
@miguelbemartin this feature is not yet supported. But it's something that's on the roadmap. If you'd like to contribute go for it!

**Comment by miguelbemartin** [comment]
Thanks for your answer. I didn't use this theme in the end as it didn't have support for pagination. If not, I would be happy to help to contribute to this repo.
